erpnext/projects/doctype/project/project.py

```diff
@@ -75,7 +75,7 @@ def validate_weights(self):
 		sum = 0
 		for task in self.tasks:
 			if task.task_weight > 0:
-				sum = sum + task.task_weight
+				sum = round(sum + task.task_weight, 3)
```

Comment:
Don't hardcode, use precision of the field instead.

`flt(sum + task.task_weight, task.precision('task_weight'))`